drivers/timer/intel_adsp_timer.c

```diff
@@ -42,12 +45,7 @@
 BUILD_ASSERT(MIN_DELAY < CYC_PER_TICK);
 BUILD_ASSERT(COMPARATOR_IDX >= 0 && COMPARATOR_IDX <= 1);
 
-#define WCTCS      (ADSP_SHIM_DSPWCTS)
-#define COUNTER_HI (ADSP_SHIM_DSPWCH)
-#define COUNTER_LO (ADSP_SHIM_DSPWCL)
-#define COMPARE_HI (ADSP_SHIM_COMPARE_HI(COMPARATOR_IDX))
-#define COMPARE_LO (ADSP_SHIM_COMPARE_LO(COMPARATOR_IDX))
-
+#define DSP_WCT_CS_TT(x)                     BIT(4 + x)
```

Comment:
isn't it defined in soc.h ?

Reply:
yeah, need to remove that from the soc.h and just declare it where it is being used.